backend/parser/src/main/java/me/codvis3D/ast/cpplisteners/CppLstnr_Initial.java

```diff
@@ -113,6 +178,135 @@ public void enterExpressionstatement(CPP14Parser.ExpressionstatementContext ctx)
 	    this.scopeStack.peek().addDataInModel(ctx.getText());
 	}
 
+	/**
+	 * Fetches a meta data of the function.
+	 *
+	 * @param      ctx   The context
+	 *
+	 * @return     return JSON object containing list of parameters, declrator ID, and scope function is from.
+	 */
+	private JSONObject fetchMetaData(CPP14Parser.FunctiondefinitionContext ctx){
```

Comment:
Not sure but think it would be better if this function was part of functionModel, fetchMetaData could be anything not just functions metaData. on the other hand ctx gives more meaning in this file.